templates-pipelines/windows-efi-installer/pipelineruns/pipelineruns.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  generateName: windows11-installer-run-
+spec:
+  params:
+    - name: winImageDownloadURL
+      value: INSERT_WINDOWS_ISO_URL
+  pipelineRef:
+    resolver: hub
+    params:
+    - name: catalog
+      value: {{ pipelines_catalog }}
+    - name: type
+      value: {{ catalog_type }}
+    - name: kind
+      value: task
+    - name: name
+      value: {{ item.pipeline_name }}
+    - name: version
+      value: {{ version }}
+  serviceAccountName: pipeline
+---
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  generateName: windows2k22-installer-run-
+spec:
+  params:
+    - name: winImageDownloadURL
+      value: INSERT_WINDOWS_ISO_URL
```

Comment:
When using this with oc create around you could make this a env var, and fill it before the call to oc create.